src/Aspire.ProjectTemplates/templates/aspire-empty/.template.config/ide.host.json

```diff
@@ -3,6 +3,13 @@
   "icon": "ide/AspireEmpty.ico",
   "displayOverviewPage": "0",
   "disableHttpsSymbol": "NoHttps",
+  "symbolInfo": [
+    {
+      "id": "AspireVersion",
+      "isVisible": true,
+      "persistenceScope": "templateGroup"
```

Comment:
So you perhaps want

"persistenceScope": "shared",
"persistenceScopeName": "aspireTemplates"

in order to sticky across all the aspire templates?